packages/client/components/ReflectionCard/ReflectionCard.tsx

```diff
@@ -14,26 +14,52 @@ import useMutationProps from '../../hooks/useMutationProps'
 import EditReflectionMutation from '../../mutations/EditReflectionMutation'
 import RemoveReflectionMutation from '../../mutations/RemoveReflectionMutation'
 import UpdateReflectionContentMutation from '../../mutations/UpdateReflectionContentMutation'
+import {PALETTE} from '../../styles/paletteV3'
 import convertToTaskContent from '../../utils/draftjs/convertToTaskContent'
 import isAndroid from '../../utils/draftjs/isAndroid'
 import isPhaseComplete from '../../utils/meetings/isPhaseComplete'
 import isTempId from '../../utils/relay/isTempId'
 import {ReflectionCard_reflection} from '../../__generated__/ReflectionCard_reflection.graphql'
+import CardButton from '../CardButton'
 import ReflectionEditorWrapper from '../ReflectionEditorWrapper'
 import StyledError from '../StyledError'
 import ColorBadge from './ColorBadge'
 import ReactjiSection from './ReactjiSection'
 import ReflectionCardDeleteButton from './ReflectionCardDeleteButton'
 import ReflectionCardRoot from './ReflectionCardRoot'
+import IconLabel from '../IconLabel'
+import useBreakpoint from '../../hooks/useBreakpoint'
+import {Breakpoint, ZIndex} from '../../types/constEnums'
+import {MenuPosition} from '../../hooks/useCoords'
+import useTooltip from '../../hooks/useTooltip'
 
 const StyledReacjis = styled(ReactjiSection)({
   padding: '0 14px 12px'
 })
 
+const SearchIcon = styled(IconLabel)({
+  color: PALETTE.SLATE_700
+})
+
+const SearchButton = styled(CardButton)<{showSearch: boolean}>(({showSearch}) => ({
+  bottom: 2,
+  color: PALETTE.SLATE_700,
+  cursor: 'pointer',
+  opacity: 1,
+  position: 'absolute',
+  right: 2,
+  visibility: showSearch ? 'visible' : 'hidden',
+  zIndex: ZIndex.TOOLTIP,
```

Comment:
It's not strictly a tooltip but hovering over the button triggers the tooltip. I went with the tooltip z-index as I didn't want to add another z-index to our collection